mbed-hal-efm32/objects.h

```diff
@@ -119,6 +113,7 @@ struct serial_s {
     DMA_OPTIONS_t dmaOptionsTX;
     DMA_OPTIONS_t dmaOptionsRX;
 #endif
+    uint32_t sleep_blocked;
```

Comment:
sleep reference counter implemented by Pollen, I'm however not convinced of its usefulness...